src/quoridor/game/game.go

```diff
@@ -38,3 +109,17 @@ func GetGame(id string) (Game, error) {
 	}
 	return game.(Game), nil
 }
+
+//AddFence add the fence on the board
+func AddFence(id string, fence Fence) (Game, error) {
+	game, err := GetGame(id)
+	if err != nil {
+		return Game{}, err
+	}
+	errFence := game.AddFence(fence)
+	if errFence != nil {
+		return Game{}, errFence
+	}
+	storage.Set(game.ID, game)
```

Comment:
You should pass a game and return it modified. Exactly as you made in "CreateGame" but without saving it